packages/site/src/pages/DesignPage.tsx

```diff
@@ -8,6 +8,8 @@ export const DesignPage = () => {
         header: {
           title: "Design",
           body: "Design elements and concepts that apply across Jobber's ecosystem",
+          imageURL:
+            "https://images.unsplash.com/photo-1482731215275-a1f151646268?q=80&w=1470&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
```

Comment:
Placeholder image to validate that a non-default image can be passed in 
![image](https://github.com/user-attachments/assets/cdedc371-286a-4fd9-aa5d-b43502c10de2)